pkg/cmd/pr/create/create.go

```diff
@@ -198,8 +204,6 @@ func NewCmdCreate(f *cmdutil.Factory, runF func(*CreateOptions) error) *cobra.Co
 	fl.StringVar(&opts.RecoverFile, "recover", "", "Recover input from a failed run of create")
 	fl.StringVarP(&opts.Template, "template", "T", "", "Template `file` to use as starting body text")
 
-	cmd.MarkFlagsMutuallyExclusive("fill", "fill-first")
```

Comment:
Having all the flag checks in the same location above will help us find this check in the future if we need to change it.